source/dotnet/AdaptiveCards.Visualizer.Wpf.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
```

Comment:
is it possible to get some tests in place?

Reply:
I'm currently adding templating function to winui visualizer; as I work on it; i'm going to add unit testing as well.